gum/arch-arm64/gumarm64writer.h

```diff
@@ -7,6 +7,9 @@
 #ifndef __GUM_ARM64_WRITER_H__
 #define __GUM_ARM64_WRITER_H__
 
+#include "gumlibc.h"
+#include "gummemory.h"
```

Comment:
Actually they are not needed at all. Idk why I put there, but once removed nothing gets broken. Removed.